rollup.config.js

```diff
@@ -201,9 +178,57 @@ export function makeBaseBundleConfig(options) {
       strict: false,
       esModule: false,
     },
-    plugins: [jsVersion === 'es5' ? typescriptPluginES5 : typescriptPluginES6, markAsBrowserBuild, nodeResolvePlugin],
+    plugins: [
+      jsVersion === 'es5' ? typescriptPluginES5 : typescriptPluginES6,
+      markAsBrowserBuildPlugin,
+      nodeResolvePlugin,
+      licensePlugin,
+    ],
     treeshake: 'smallest',
   };
 
   return deepMerge(sharedBundleConfig, isAddOn ? addOnBundleConfig : standAloneBundleConfig);
 }
+
+export function makeMinificationVariants(existingConfigs) {
+  const newConfigs = [];
+
+  // ensure we've got an array of configs rather than a single config
+  existingConfigs = Array.isArray(existingConfigs) ? existingConfigs : [existingConfigs];
+
+  existingConfigs.forEach(existingConfig => {
+    const { plugins } = existingConfig;
+
+    // The license plugin has to be last, so it ends up after terser. Otherwise, terser will remove the license banner.
+    assert(
+      getLastElement(plugins).name === 'rollup-plugin-license',
+      `Last plugin in given options should be \`rollup-plugin-license\`. Found ${getLastElement(plugins).name}`,
+    );
+
+    const minificationVariants = [
```

Comment:
Why are these `minificationVariants` if only one outputs `min.js`?

Reply:
I was thinking about it as minification-yes and minification-no, i.e., they are varying on the "should I minify" axis. I debated `minificationOptionsVariants`, also. Do you think that's clearer? Or can you think of a better name? I'll admit I struggled with this one a bit. #thehardestprobleminprogramming

Reply:
IMO `bundleVariants` expresses good clarity here.